index.html

```diff
@@ -122,6 +122,33 @@ <h2>A project for first-time contributions</h2>
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
 
+         <!-- ________ *Your_Name* card START ________  -->
+         <div class="card">
+          <p class="name">Rekt46</p>
+          <p class="contact">
+            <i class="fab fa-x-twitter"></i>
+            <a href="https://twitter.com/Rektama46olan" target="_blank">Rekt46</a>
+            <i class="fab fa-github"></i>
+            <a href="https://github.com/Rektama46OLAN" target="_blank">Rekt46</a>
+          </p>
+          <p class="about">As someone new to software, I'm happy to discover this project.</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://www.youtube.com/watch?v=-2OW72rBQr8" target="_blank" title="Best 6 Free Web Design Software to Help You Build a Website">Resource 1</a>
```

Comment:
Please change 'Resource 1' to a more speaking title, same for 'Resource 2' and 'Resource 3' as they are displayed on the user's screen.